config.json

```diff
@@ -189,6 +189,16 @@
       ]
     },
     {
+      "uuid": "4a83a72c-db0a-45b6-b77c-1949cb24fbae",
+      "slug": "sublist",
+      "difficulty": 4,
+      "core": false,
+      "unlocked_by": null,
+      "topics": [
```

Comment:
Another topic could be `Arrays`.

I'm not doing a clear distinction between `Lists` and `Arrays` in JavaScript. For me, it's more or less like: `Arrays` is for the language itself (`[]`, `[1, 2, 3]`) and `Lists` is more about the programming concept, the data structure, manipulation of `Arrays` (in case of JS). As I said, it's not clear enough for me, so I'm not sure this explanation is clear for others, sorry.